xbmc/pictures/GUIWindowSlideShow.cpp

```diff
@@ -737,7 +737,7 @@ bool CGUIWindowSlideShow::OnAction(const CAction &action)
 {
   switch (action.GetID())
   {
-  case ACTION_SHOW_CODEC:
+  case ACTION_PLAYER_PROCESS_INFO:
     {
       CGUIDialogPictureInfo *pictureInfo = (CGUIDialogPictureInfo *)g_windowManager.GetWindow(WINDOW_DIALOG_PICTURE_INFO);
```

Comment:
imo neither ACTION_SHOW_CODEC nor ACTION_PLAYER_PROCESS_INFO is correct here. Seems that slide show simply misuses an existing action with completely unrelated semantics to display a picture info dialog.